### DISCUSSION TITLE
The visual difference between the image transformed from kspace and 'reconstruction_rss' from the h5 file.

### DISCUSSION THREAD

**Discussion by YSongxiao** [facebookresearch/fastMRI#327] [Q&A]
Hello, when I try to use fastmri's ifft operation to transform 'kspace' data to image domain (single coil knee dataset), I found that the fully sampled MR image is very noisy. However, when I just load the image from the 'reconstruction_rss', the image is clean. I wonder whether there is any wrong with my code or the image reconstructed from kspace data is not the same with 'reconstruction_rss'.

```
import fastmri
from fastmri.data import transforms as T
import h5py
import matplotlib.pyplot as plt

file_name = "/singlecoil_val/file1000031.h5"
hf = h5py.File(file_name, 'r')
volume_kspace = hf['kspace'][()]
slice_kspace = volume_kspace[20]
slice_kspace2 = T.to_tensor(slice_kspace)      # Convert from numpy array to pytorch tensor
slice_image = fastmri.ifft2c(slice_kspace2)           # Apply Inverse Fourier Transform to get the complex image
slice_image_abs = fastmri.complex_abs(slice_image)
```

**Comment by mmuckley** [comment]
Hello @YSongxiao, that does seem a bit odd. Could you post some images here? I'm also curious about a comparison to the multicoil version of `file1000031.h5`.

**Reply by YSongxiao** [reply]
Hello @mmuckley , thank you for your reply. I used the 20th slice of file1000031.h5 as an example. There are also the same issues happening to the other files in single-coil data.
This is reconstructed from kspace data of multi-coil file. It looks okay.
![multi_file1000031_slice20_rss_no_norm](https://github.com/facebookresearch/fastMRI/assets/55771652/c075f49f-ec02-467e-b5cc-a77cd85ea653)
This is reconstructed from kspace data of single-coil file. It is quite noisy. If you download it and increase the brightness, it will be more obvious.
![single_file1000031_slice20_no_norm](https://github.com/facebookresearch/fastMRI/assets/55771652/187e36e2-0a5a-4ac1-a0a3-306c5e75c338)
This is loaded from h5 file of single-coil version. It looks also okay.
![single_file1000031_slice20_reconstruction_rss_no_norm](https://github.com/facebookresearch/fastMRI/assets/55771652/2f61f112-f618-4ef4-908d-b312b982fe93)

**Reply by mmuckley** [reply]
Hmm, yeah it does look pretty clean. I don't think I modified the singlecoil files when we did the data split, though. I'll look at it a bit more.

**Reply by YSongxiao** [reply]
Thanks, I will wait for your further response.
